scripts/update_changelog.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+#
+# Run this script from the top level directory of the trillian repo e.g.
+# with scripts/update_changelog.sh.
+#
+# GOPATH must be set.
+
+set +e
+d=${GOPATH[0]}
+PATH=$PATH:${d}/bin
```

Comment:
Might be safer to reverse this so the `github-release` invocation below should always get the freshly installed version, not whatever might be lurking in `PATH`

Reply:
Moot.